components/brave_adaptive_captcha/get_adaptive_captcha_challenge_unittest.cc

```diff
@@ -56,12 +55,10 @@ class GetAdaptiveCaptchaChallengeTest : public testing::Test {
     SignalUrlLoadCompleted();
   }
 
-  std::unique_ptr<GetAdaptiveCaptchaChallenge> GetChallenge() {
-    return std::move(get_challenge_);
-  }
-
  protected:
   network::TestURLLoaderFactory test_url_loader_factory_;
+  api_request_helper::APIRequestHelper api_request_helper_;
+  std::unique_ptr<GetAdaptiveCaptchaChallenge> get_challenge_;
```

Comment:
Just reverting some unittest changes so they don't need to make extra considerations for the relationship between api_request_helper and data decoder